services/src/main/scala/cromwell/services/metadata/impl/FileDeletionStatus.scala

```diff
@@ -0,0 +1,14 @@
+package cromwell.services.metadata.impl
+
+sealed trait FileDeletionStatus
+
+object FileDeletionStatus {
+
+  lazy val FileDeletionStatusValues = Seq(InProgress, Succeeded, Failed)
```

Comment:
Is this used anywhere.

Also ToL: I wonder which we'll end up using first: [Enumeratum](https://github.com/lloydmeta/enumeratum) or [Scala 3](https://dotty.epfl.ch/docs/reference/enums/enums.html)

Reply:
Oh. No. I had some code which was using it, but it was removed during clean up. I remove it.